aws-lc-rs/src/aead.rs

```diff
@@ -72,21 +72,17 @@ use key_inner::KeyInner;
 use std::mem::MaybeUninit;
 use std::ops::RangeFrom;
 
-mod aes;
 mod aes_gcm;
-mod block;
 mod chacha;
 pub mod chacha20_poly1305_openssh;
-mod cipher;
 mod key_inner;
 mod nonce;
 pub mod nonce_sequence;
 mod poly1305;
 pub mod quic;
 
 pub use self::{
-    aes_gcm::{AES_128_GCM, AES_128_KEY_LEN, AES_256_GCM, AES_256_KEY_LEN},
```

Comment:
There is value to exposing `AES_128_KEY_LEN` and `AES_256_KEY_LEN` and possibly a `AES_BLOCK_LEN` to consumers, but they shouldn't be exposed from the `aead` module.

Reply:
We could potentially move these constants to `cipher::aes`.